script.module.clouddrive.common/clouddrive/common/ui/addon.py

```diff
@@ -353,29 +369,40 @@ def _process_items(self, items, driveid):
                 if 'thumbnail' in item:
                     list_item.setIconImage(item['thumbnail'])
                     list_item.setThumbnailImage(item['thumbnail'])
```

Comment:
`setThumbnailImage, setIconImage` is deprecated; please use setArt()

https://codedocs.xyz/xbmc/xbmc/group__python__xbmcgui__listitem.html#gad3f9b9befa5f3d2f4683f9957264dbbe

Reply:
Will be changed